group.go

```diff
@@ -367,6 +367,12 @@ func (g *Group) View() string {
 		// append an empty line, and the footer (usually the help).
 		parts = append(parts, "", s)
 	}
+	if len(parts) > 0 {
+		// XXX: Trim suffix spaces from the last part as it can accidentally
+		// scroll the view up on some terminals when we right to the bottom
+		// rightmost corner cell.
```

Comment:
```suggestion
		// Trim suffix spaces from the last part as it can accidentally
		// scroll the view up on some terminals (like Apple's Terminal.app)
		// when we right to the bottom rightmost corner cell.
```